test.js

```diff
@@ -1,16 +1,21 @@
-const quote=require('./index.js');
+const quote = require('./index.js');
+const data = require('./data/data.json');
+const assert = require('assert');
 
-var Quote = quote.getQuote();
+describe("Test Suite", () => {
+    it("test for Duplicacy of quotes", () => {
```

Comment:
`describe` blocks should match the thing you're testing. `it` blocks should match what behavior you're testing for. The assertions inside the it should describe how you're validation the behavior.

for this case:

```js
describe("data.json", () => {
    it("should not contain any duplicate quotes", () => {
```
Now, it reads like a sentence. Later, if we add a few more tests it'll be more readable.